src/votekit/graphs/ballot_graph.py

```diff
@@ -185,9 +190,13 @@ def fix_short_ballot(self, ballot: list, candidates: list) -> list:
 
         return ballot + list(missing)
 
-    def label_cands(self, candidates):
+    def label_cands(self, candidates,
+                    to_display: Optional[Callable] = all_nodes):
```

Comment:
Here's the issue with the type hinting as there's technically None being passed when this argument is not included. Changing it to `to_display: Optional[Union[Callable, None]]` I think will fix the linting issue. 

Separately, can you explain why this is a callable, what is the user supposed to input if they want to to only display a subset of the nodes? Is it a function that returns True/False for a given node?

Reply:
Yeah, the callable should take in the graph and a node and return True if that node should be displayed. I just added an edit to make sure the graph was an argument to these callables.

Reply:
Ah looks like the thing that fixed it was just using `Callable`. Now the only thing failing is `test_interval_from_params`, which is that non-deterministic test. Can you re-run it for me @jgibson517 ? Thanks!

Reply:
👍